src/Employer/Employer.Web/Orchestrators/VacancyViewOrchestrator.cs

```diff
@@ -43,6 +43,11 @@ public async Task<DisplayVacancyViewModel> GetFullVacancyDisplayViewModelAsync(V
         {
             switch (vacancy.Status)
             {
+                case VacancyStatus.Approved:
```

Comment:
Per Mark's comment on V2 channel, this should show as Live.

Reply:
I'm not sure the status ever gets set to `Approved` at the moment so doubt it will drop into here anyway.  It will in future though when we have deferred live dates.